### PR TITLE
Updated link + fixed full demo file

### PR DESCRIPTION
**Pull request by ghost**
- Updated link for 'create api keys'
- Fixed 'Full demo file' at the end

### CODE REVIEW

File: docs/streams/getting-started-streams.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This article is an introduction to using streams with [Macrometa SDKs](../sdks/i
 ## Prerequisites
 
 - Access to a Macrometa account with sufficient permissions to create streams.
-- Have your API Key ready. For more information, refer to [Create API Keys] (../account-management/api-keys/create-api-keys)
+- Have your API Key ready. For more information, refer to [Create API Keys] (https://macrometa.com/docs/account-management/api-keys/create-api-keys).
```
Comment (diana-macrometa) [inline docs/streams/getting-started-streams.md]: We want to keep this as a relative link.

File: docs/streams/getting-started-streams.md [hunk 2]
```diff
@@ -398,7 +398,7 @@ warnings.filterwarnings("ignore")
 # Connect to GDN.
 URL = "gdn.paas.macrometa.io"
 GEO_FABRIC = "_system"
-API_KEY = "xxxxx" # Change this to your API key
+API_KEY = "xxxx" # Change this to your API key
```
Comment (diana-macrometa) [inline docs/streams/getting-started-streams.md]: Why?

File: docs/streams/getting-started-streams.md [hunk 3]
```diff
@@ -426,32 +426,38 @@ def createStream():
 def sendData():
     """ This function sends data through a stream """
     producer = client.create_stream_producer(demo_stream, local=is_local)
-    for i in range(10):
-        msg1 = "Persistent Hello from " + "("+ str(i) +")"
-        print("Stream: ", msg1)
-        producer.send(msg1)
+    while True:
```
Comment (diana-macrometa) [inline docs/streams/getting-started-streams.md:429]: Why did you change the `for` loop to `while`?

File: docs/streams/getting-started-streams.md [hunk 4]
```diff
@@ -426,32 +426,38 @@ def createStream():
...
+        print("\nListening for message...")
+        if user_input == '0':
+            break
         m1 = json.loads(subscriber.recv())  # Listen on stream for any receiving messages
         msg1 = base64.b64decode(m1["payload"]).decode('utf-8')
-        print(F"Received message '{msg1}' id='{m1['messageId']}'") # Print the received message
+        print(F"Received message: '{msg1}'") 
+    #Output the ID of the received message
+        #print(F"Message ID:'{m1['messageId']}'")
```
Comment (diana-macrometa) [inline docs/streams/getting-started-streams.md]: Why is this print statement commented out?

**Review by diana-macrometa — CHANGES_REQUESTED** [review]

**Review by diana-macrometa — APPROVED** [review]
Looks good!

**Review by ghost — COMMENTED** [review]
lowercase old/new
